src/source/PeerConnection/SessionDescription.c

```diff
@@ -642,9 +642,8 @@ STATUS deserializeRtcIceCandidateInit(PCHAR pJson, UINT32 jsonLen, PRtcIceCandid
     tokenCount = jsmn_parse(&parser, pJson, jsonLen, tokens, ICE_CANDIDATE_INIT_TOKENS);
     CHK(tokenCount > 1, STATUS_INVALID_API_CALL_RETURN_JSON);
     CHK(tokens[0].type == JSMN_OBJECT, STATUS_ICE_CANDIDATE_INIT_MALFORMED);
-    CHK(tokenCount == ICE_CANDIDATE_INIT_TOKENS, STATUS_ICE_CANDIDATE_INIT_MALFORMED);
```

Comment:
Why don't we need to track ICE_CANDIDATE_INIT_TOKENS here?

Reply:
The other values are actually [optional](https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidateInit)! When I wrote it the first time I just asserted it was the length that Chromium sent. 

It should have no effect, but I just wanted to remove the overly assertive check.